src/Modules/OrchardCore.Commerce/Services/RegionConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.Extensions.Options;
+using OrchardCore.Commerce.Models;
+using OrchardCore.Entities;
+using OrchardCore.Settings;
+
+namespace OrchardCore.Commerce.Services;
+
+public class RegionSettingsConfiguration : IConfigureOptions<RegionSettings>
+{
+    private readonly ISiteService _siteService;
+
+    public RegionSettingsConfiguration(ISiteService siteService) =>
+        _siteService = siteService;
+
+    public void Configure(RegionSettings options)
+    {
+        var settings = _siteService
+            .GetSiteSettingsAsync()
+            .GetAwaiter()
+            .GetResult()
+            .As<RegionSettings>();
+
+        options.AllowedRegions = settings.AllowedRegions;
```

Comment:
Can you create an recipe that initializes this to 3-5 countries? It should not be included in the setup recipe but available for unit testing.

Reply:
I also made region settings exportable.